src/process/WindowsLokinetProcessManager.cpp

```diff
@@ -18,7 +18,7 @@ WindowsLokinetProcessManager::WindowsLokinetProcessManager()
 bool WindowsLokinetProcessManager::doStartLokinetProcess()
 {
     // try searching one level up from CWD
-    bool success = success = QProcess::startDetached("net start lokinet");
+    bool success = QProcess::startDetached("net start lokinet");
     if (success)

```

Comment:
`if(not success)`

Reply:
right i got that mixed up with the plain old `system(3C)`